### DISCUSSION TITLE
Washington DC has taken ctrl+/

### DISCUSSION THREAD

**Discussion by jimfricker** [arnoudkooi/SN-Utils#487] [General]
Not sure if anyone else has noticed. Sorry if this has already been discussed, but Washington DC has taken the ctrl+/ keyboard shortcut so it no longer works with SNUtils. I am not a fan of just "/" because I often use it when in a form and it just types it in the field that has focus. Anyone else found this?

**Comment by arnoudkooi** [comment]
Correct :( 
As an alternative, ALT-/ is [added](https://twitter.com/sn_utils/status/1753558008662049231) to Windows
Does that work for you?

(Mac shifted from CMD-/ tor CTRL-/)

**Reply by jimfricker** [reply]
yes thanks, that works for me

**Reply by codaroma** [reply]
I have decided to use the "Off - can set custom shortcut" option. I have then used the browser's "Manage Extension Shortcuts" to map it to an obscure key combination and I have then used AutoHotkey to map the CapsLock key to this obscure combination. I used to hate my CapsLock key. Now I love it.
